tests/tools/oesign/test-enclave/enclave/enc.c

```diff
@@ -52,3 +53,61 @@ bool is_test_signed()
 
     return is_test_signed;
 }
+
+oe_result_t is_kss_extendedids_match(
```

Comment:
Maybe 'check_kss_extended_ids' is a better name?

Reply:
Done